### ISSUE TITLE
Pages have no content without js

### ISSUE THREAD

**Issue by sharpobject** [NotWarlock/Demonology#102]
For normal workflows like retrieving a page using wget then browsing it while offline, the user ends up with a totally blank page. The totally blank page is identical for all pages of the app, because there is only one URI and the content is indexed by anchor rather than by URI.

**Comment by sergiomasellis** [comment]
Hey @sharpobject thank you for reporting this. I will look into the possibility of adding this as an enhancement. It would mean creating duplicate pages as we don't have a back end currently and everything is hosted on github. 

For the sake of the guide writer who have little to no html experience my goal was to make it as simple as possible to create these pages. Each page excludes <head><body> and everything that entails creating the page. Those fragments are then loading as html via ajax requests which is why offline mode pages currently do not work.

In summary I can look into how I can allow these power user enhancements.
